src/main/java/org/jetbrains/research/refactorinsight/data/diff/DiffRequestGenerator.java

```diff
@@ -48,6 +48,8 @@ public List<RefactoringLine> getMarkings() {
     return lineMarkings;
   }
 
+  public abstract boolean containsElement(int lineNumber, int textOffset, boolean isRight);
```

Comment:
`isRight` means that we search for an element on the right side of the diff?

Reply:
Yes